package.json

```
@@ -33,6 +33,7 @@
     "compile:node": "webpack --progress --env target=node",
     "compile:types": "tsc --emitDeclarationOnly --declaration",
     "compile:browser": "webpack --progress --env target=web",
+    "ci": "npm run lint && npm run check:types && npm run test",
```

I wonder if you only check the types, why you don't check the lint as well instead of make the corrections?

```suggestion
    "ci": "npm run lint:check && npm run check:types && npm run test",
```